lego-mindstorms-pieces.py

```diff
@@ -174,11 +178,38 @@ def order(shop=None, browser=None, lego_set=None, order_list=None, username=None
 
     order_list = order_list.split(',')
 
-    shop_url = 'https://wwwsecure.us.lego.com/{shop}/service/replacementparts/order'.format(shop=shop)
-    browser = Chrome() if browser == 'chrome' else Firefox()
+    print("Using Selenium version : ",webdriver.__version__)
+
+    shop_url = 'https://wwwsecure.us.lego.com/{shop}/service/replacementparts/sale'.format(shop=shop)
+
+    print("Browser URL : ",shop_url)
+
+    ## detect browser choice
+    if browser == 'chrome':
+        opts = ChromeOptions()
+        # With selenium version above this one, chrome is closed at the end without the "quit()" method !
```

Comment:
Please wrap the line before 80 characters are reached, and don't use a space before the exclamation mark.